client/src/components/forms/FormError.js

```diff
@@ -0,0 +1,13 @@
+const FormError = (props) => {
+    return (
+        <div>
+            {props.error &&
+                <span className="error">
+                    <strong>{props.errorMsg}</strong>
+                </span>
+            }
+        </div>

```

Comment:
i think a fragment here will suffice (instead of a div).

in the tailwind css file, the error styles is already defined as `.form span`  so that file will also need to be updated.

not sure if we need the `<strong>` tag from a stylistic perspective, but that's not a blocker.